client/src/app/ui/modules/media-upload-content/media-upload-content.component.ts

```diff
@@ -19,7 +27,20 @@ export interface UploadSuccessEvent {
 @Component({
     selector: `os-media-upload-content`,
     templateUrl: `./media-upload-content.component.html`,
-    styleUrls: [`./media-upload-content.component.scss`]
+    styleUrls: [`./media-upload-content.component.scss`],
+    standalone: true,
+    imports: [
+        CommonModule,
+        FormsModule,
+        ReactiveFormsModule,
+        MatIconModule,
+        MatInputModule,
+        MatFormFieldModule,
+        PipesModule,
+        SearchSelectorModule,
+        FileUploadModule,
+        OpenSlidesTranslationModule
```

Comment:
https://ngx-translate.org/getting-started/migration-guide/#updating-your-standalone-components

```suggestion
        TranslatePipe
```

Reply:
Done, use the TranslatePipe.